wp-content/themes/access/single-programs.php

```diff
@@ -111,50 +111,57 @@
   array_push($context['schema'], $special_announcement);
 }
 
-if($program->validateFaq() === true) {
+/**
+* The array $questions has a set of elements that are the questions to be added
+* to the $faq variable which will be added to the schema as the `FAQPage`
+* section.
+*/
+$questions = [
```

Comment:
@theasteve how about adding these schemas to the `programs.php` controller? It will be easier to share these schemas across the site if they are in the controller.